src/libertem/web/notebook_generator/.gitignore

```diff
@@ -0,0 +1,3 @@
+dask-worker-space/
+__pycache__/
+.ipynb_checkpoints/
```

Comment:
These should all be already ignored by the main gitignore, right? This probably comes from the move from your own repository to here

Reply:
Yes, there is no need for this `.gitignore`